client/blocks/inline-help/contextual-help.js

```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
+			title: translate( 'Jetpack Plans' ),
+			description: translate(
+				'Learn about the free Jetpack plugin, its benefits, and the useful capabilities and features that a Jetpack plan unlocks.'
+			),
 		},
 	],
 	'post-editor': [
 		{
 			link: 'http://en.support.wordpress.com/editors/',
 			post_id: 3347,
-			title: 'The Visual Editor and the HTML Editor',
-			description:
+			title: translate( 'The Visual Editor and the HTML Editor' ),
+			description: translate(
 				'When creating a post or page on your WordPress.com blog, you have two editing modes ' +
-				'available to you: the Visual Editor and the HTML Editor.',
+					'available to you: the Visual Editor and the HTML Editor.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/visual-editor/',
 			post_id: 3644,
-			title: 'The Visual Editor',
-			description:
+			title: translate( 'The Visual Editor' ),
+			description: translate(
 				'The visual editor provides a semi-WYSIWYG (What You See is What You Get) content editor that ' +
-				'allows you to easily create, edit, and format your content in a view similar to that of a word processor.',
+					'allows you to easily create, edit, and format your content in a view similar to that of a word processor.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/xml-rpc/',
 			post_id: 3595,
-			title: 'Offline Editing',
-			description:
-				'Learn how to create and edit content for your WordPress.com site even without being connected to the internet!',
+			title: translate( 'Offline Editing' ),
+			description: translate(
+				'Learn how to create and edit content for your WordPress.com site even without being connected to the internet!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/adding-users/',
-			title: 'Inviting Contributors, Followers, and Viewers',
-			description:
-				'Invite contributors, followers, and viewers to collaborate with others and grow your audience!',
+			title: translate( 'Inviting Contributors, Followers, and Viewers' ),
+			description: translate(
+				'Invite contributors, followers, and viewers to collaborate with others and grow your audience!'
+			),
 		},
 	],
 	reader: [
 		{
 			link: 'http://en.support.wordpress.com/reader/',
 			post_id: 32011,
-			title: 'The Reader: An Overview',
-			description:
+			title: translate( 'The Reader: An Overview' ),
+			description: translate(
 				'Read posts from all the sites you follow — even ones that aren’t on WordPress.com! ' +
-				'Discover great new reads and keep track of your comments and replies in one convenient place.',
+					'Discover great new reads and keep track of your comments and replies in one convenient place.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/following/',
-			title: 'Follow Blogs',
-			description:
-				'When you follow a blog on WordPress.com, new posts from that site will automatically appear in your Reader.',
+			title: translate( 'Follow Blogs' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **59** times:
`translate( 'Blogs I Follow' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).